projects/components/src/stepper/stepper.component.ts

```diff
@@ -151,6 +160,14 @@ export class StepperComponent implements AfterContentInit {
     }
   }
 
+  public next(): void {
+    this.stepper?.next();
+  }
+
+  public previous(): void {
+    this.stepper?.previous();
+  }
+
```

Comment:
Getting confused amongst these and the ones which are inside `StepperTabComponent`. Can you elaborate diff usages?

Reply:
https://github.com/hypertrace/hypertrace-ui/pull/1969#discussion_r1057474260

For making it easier for the custom controls to perform do common actions like next and prev, we expose the methods `next` and `previous` in the `ht-stepper-tab-controls` component. So the consumer can get the reference to the component by adding a `#variable`:
```html
<ht-stepper-tab-controls #controls></ht-stepper-tab-controls>
```

They can access methods exposed by the `StepperTabControlsComponent` by doing:
```html
<button (click)="controls.previous()"></button>
```

Reply:
Do we really need the methods which are defined here?

Reply:
If we make the `stepper` private, we'll need to expose these methods.